packages/plugins/rubygem-foreman_webhooks/rubygem-foreman_webhooks.spec

```diff
@@ -33,8 +33,8 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 6.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 7.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.15.1
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 5.0.0
```

Comment:
this is not going to work, we have builder 8 now, can you please drop the upper bound, this seems to work fine for other plugins

Reply:
Removed, thanks.